app/main.py

```diff
@@ -1,3 +1,5 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    small_phrase = phrase.lower()
+    small_letter = letter.lower()
```

Comment:
The conversion to lowercase is correctly done for `letter`. Ensure that `letter` is a single character string to avoid any unexpected behavior.